content/developer/reference/extract_api.rst

```diff
@@ -1,521 +1,775 @@
 
-The extract API use the JSON-RPC2_ protocol. The diffent routes are located at the following address: **https://iap-extract.odoo.com**.
+The extract API uses the JSON-RPC2_ protocol; its endpoint routes are located at
+`https://iap-extract.odoo.com`.
 
-Expected successful flow
-------------------------
+Version
+-------
 
-1. Call :ref:`webservices/extract_api/invoice_parse` to submit your invoices (one call for each invoice). On success, you receive a `document_id` in the response.
-2. You then have to regularly poll :ref:`webservices/extract_api/invoice_get_results` to get the document's parsing status.
-3. Once the result received, you can validate it by calling :ref:`webservices/extract_api/invoice_validate` and sending the expected values.
-   This step is optional but greatly helps the system to improve.
+The version of the Extract API is specified in the route.
 
-These 3 routes are detailed in this :ref:`section <webservices/extract_api/routes>`. The HTTP POST method should be used for all of them. A python implementation of the full flow
-can be found :download:`here <extract_api/implementation.py>` and a token for integration testing is provided in the :ref:`integration testing section <webservices/extract_api/integration_testing>`.
+The latest versions are:
+    - invoices: 122
+    - expenses: 132
+    - applicant: 102
 
-.. _webservices/extract_api/routes:
+Flow
+----
 
-Routes
-------
+The flow is the same for the three services.
+
+#. | Call :ref:`/parse <extract_api/parse>` to submit your invoices (one call for each
+     invoice). On success, you receive a `document_uuid` in the response.
+#. | You then have to regularly poll :ref:`/get_result <extract_api/get_result>` to get the
+     document's parsing status.
+   | Alternatively, you can provide a `webhook_url` at the time of the call to
+     :ref:`/parse <extract_api/parse>` and you will be notified (via a POST request) when the
+     result is ready.
+#. | Once the result is received, you can validate it by calling
+     :ref:`/validate <extract_api/validate>` and sending the expected values.
+   | This step is optional but greatly helps the system to improve.
+
+The HTTP POST method should be used for all of them. A python implementation of the full flow for
+invoices can be found :download:`here <extract_api/implementation.py>` and a token for integration
+testing is provided in the
+:ref:`integration testing section <latestextract_api/integration_testing>`.
 
-.. _webservices/extract_api/invoice_parse:
 
-``/iap/invoice_extract/parse``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Parse
+=====
 
-Description
-***********
+Request the processing of a document from the OCR. The route will return a `document_uuid`
+(it replaces `document_id`, which is deprecated); you can use it to obtain the result of your
+request.
 
-Request a processing of the document from the OCR. The route will return a `document_id` you can use to obtain the result of your request.
+.. _extract_api/parse:
```

Comment:
I'm unsure if you intend to have this anchor target the "Routes" section or the parent section "Parse"? In the latter case, you must move the anchor before the section title.

Reply:
It is meant for the routes, since this is what it is referring to when used in the text.